src/main.py

```diff
@@ -0,0 +1,5 @@
+from bitmessagemain import main
```

Comment:
Where is this module used? And why it has such common name?

Reply:
As we are using buildozer which is a tool for creating application packages easily for mobile client and we need to have main.py file as buildozer will look for it,as for PyBitmessage we need to start all the necessary threads at the very starting so I have to explicity call bitmessage main inside this main module.